### DISCUSSION TITLE
Vertex shaders

### DISCUSSION THREAD

**Discussion by Visioneery** [bloc97/Anime4K#189] [General]
Hi! 
I am completely new to opengl, glsl shaders etc. 
I just started learning and created project where I wanted to apply one of your shaders e,g Anime4K_Deblur_DoG using OpenTK library, but if I understand correctly Anime4K_Deblur_DoG is a fragment shader. Is Anime4k application using any vertex shaders when processing an image? are they stored somewhere?
Best regards

**Comment by bloc97** [comment]
This project does not use any vertex shaders. I'm assuming that the video plane is rendered with a simple quad that is stretched to fill the screen in mpv, so no vertex transformations are necessary.
